README.md

```diff
@@ -1,10 +1,22 @@
 # Push Notifications
-Send notifications to your phone or other devices, currently this only supports [Pushbullet](https://www.pushbullet.com/).
+Send notifications to your phone or other devices, currently supports [Pushbullet](https://www.pushbullet.com/) and [Pushover](https://pushover.net/).
 ## Pushbullet
 You need to provide a pushbullet api key to use this service.
 Go [here](https://www.pushbullet.com/#settings) and click create new token or use your existing one.
 ![create-new-pushbullet](imgs/create-new-pushbullet.png)
 
 The generated key will be in the format `o.Dorf43jdDIepfKeroPewfjeIUHJ4MrOP`.
 
-Open your client and paste the key into the `Pushbullet` config field of `Push Notifications` to enable phone notifications.
\ No newline at end of file
+Open your client and paste the key into the `Pushbullet` config field of `Push Notifications` to enable phone notifications.
+## Pushover
+You need to provide both a pushover user key and api token to use this service.
+
+Go [here](https://www.pushbullet.com/), login and your user key will be in the top right of the page.
```

Comment:
This should be pushover

Reply:
Must have forgotten to change it in the copy and paste good spot!